examples/compactCells.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018, 2020 Uber Technologies, Inc.
+ * Copyright 2018-2021 Uber Technologies, Inc.
```

Comment:
Nitpicking, but this should be `2018, 2020-2021` since no changes to the file occurred in 2019.